soh/soh/z_message_OTR.cpp

```diff
@@ -63,14 +96,12 @@ MessageTableEntry* OTRMessage_LoadTable(const char* filePath, bool isNES) {
             table[file->messages.size()].msgSize = kaeporaMsgSize;
         }
 
-        table[i].textId = file->messages[i].id;
-        table[i].typePos = (file->messages[i].textboxType << 4) | file->messages[i].textboxYPos;
-        table[i].segment = file->messages[i].msg.c_str();
-        table[i].msgSize = file->messages[i].msg.size();
+        SetMessageEntry(table[i], file->messages[i]);
 
         if (isNES && file->messages[i].id == 0xFFFC)
             _message_0xFFFC_nes = (char*)file->messages[i].msg.c_str();
     }
+    OTRMessage_LoadCustom("custom/" + filePath.substr(0, filePath.find_last_of('/')) + "/*", table, file->messages.size() + 1);
```

Comment:
We will want to use the `custom/` prefix in a specific way that won't necessarily be compatible with this later on, so for now change this to use `override/` as a prefix for the overriding vanilla text method.